cmd/finch/nerdctl.go

```diff
@@ -45,7 +45,7 @@ type nerdctlCommandCreator struct {
 
 type (
 	argHandler     func(systemDeps NerdctlCommandSystemDeps, fc *config.Finch, args []string, index int) error
-	commandHandler func(systemDeps NerdctlCommandSystemDeps, fc *config.Finch, cmdName *string, args *[]string) error
+	commandHandler func(systemDeps NerdctlCommandSystemDeps, fc *config.Finch, cmdName *string, args *[]string, inspectType *string) error
```

Comment:
This is an interesting mechanism to enable the specific `handleDockerCompatInspect` to affect operation of the parent process.
Perhaps, in the future, we can generalize this var type so that it is more useful for any of the `cmdHandler` functions, such as a `map[string]string`.

Reply:
may be if required, this one will be removed once we fix it upstream.